test/images/resource-consumer/README.md

```diff
@@ -0,0 +1,66 @@
+# Resource Consumer container for testing
+
+Resource Consumer is a tool for testing kubernetes autoscaling.
+It can help with tests for:
+- cluster size autoscaling
+- horizontal autoscaling of pod - changing the size of replication controller
+- vertical autoscaling of pod - changing its resource limits
+
+image: gcr.io/google_containers/resource_consumer:beta
```

Comment:
Please extend this line. It not clear for me why it is here. For example: Docker image of Resource Consumer can be found in Google Container Registry as ...